### ISSUE TITLE
Custom Animated Windows Cursor have incorrect colors

### ISSUE THREAD

**Issue by RadarNyan** [rustdesk/rustdesk#4652]
### Bug Description

See screenshot.

### How to Reproduce

Set a custom animated cursor and connect?

### Expected Behavior

To have correct cursor displayed, or have an option to use a plain arrow ignoring remote cursor.

### Operating system(s) on local side and remote side

Android -> Windows

### RustDesk Version(s) on local side and remote side

1.1.9 -> 1.1.9

### Screenshots

![Screenshot_RustDesk_20230616-003359_1](https://github.com/rustdesk/rustdesk/assets/2091529/e446c7a8-9bc3-431c-b4cd-7648edc4acb4)

### Additional Context

The cursor in use is animated (.ani), not sure if it has something to do with the weird colors.

My custom cursor is made by Pixiv user rasa: https://www.pixiv.net/users/263205
Download link and password are in this post: https://www.pixiv.net/artworks/58815459
The cursor in-use in the screenshot is No.43 from the set.

**Comment by RadarNyan** [comment]
Here's a single file for "Normal Selection" if you don't wanna download the entire set.

And looking close at it, it's just not the colors are wrong, the cursor image have weird clippings (left colum pixels are cut-off and appears on right side?)

[Shimakaze_Normal_Selection.zip](https://github.com/rustdesk/rustdesk/files/11760228/Shimakaze_Normal_Selection.zip)

**Comment by rustdesk** [comment]
wontfix in current plan.

**Comment by rustdesk** [comment]
In nightly build, you will see a dot.

**Comment by RadarNyan** [comment]
Things I've tried: (using nightly build 52ce3dd)

1. Switch to Windows 10 on server side - same result.
2. Install nightly build for android - same result. (*see below)
3. Uninstall and run nightly build for windows - same result with many more issues: unable to maintain connection (Connection Error: Closed manually by the peer) immediately after successfully connected (so I got one frame on the phone which I can see the cursor problem doesn't change)

* It seems this problem is on the Windows server side fail to capture the correct cursor (on launch?), since the cursor didn't change on Android client side until I restart the Windows server side.

Also the nightly build had some changes that I dislike, for example:
1. Password to must contain Upper, lower, number - I get it needs to be secure, but I have my own password scheme and every time a website/service forces their own scheme on me just makes it harder for me to remember and have to resort to writing it down or store it somewhere, which isn't really a good idea for security. I understand those are good "recommendations" for a somewhat secure password, but would you please don't force it?
2. Unable to turn off IP whitelist feature without clearing the list - this just makes it harder for debugging and I don't understand why would you design it like this. Also, it breaks the universal design language of what a checkbox supposed to do. During my test, when whitelist is enabled I won't be able to connect (Login Error: Your ip is blocked by the peer) when both server side and client side are using nightly build, the ip address is correct as it works fine with the stable release.
